components/timezone/src/lib.rs

```diff
@@ -81,17 +82,18 @@
 //! the metazone based on a certain local datetime:
 //!
 //! ```
-//! use icu_calendar::DateTime;
-//! use icu_timezone::CustomTimeZone;
-//! use icu_timezone::GmtOffset;
-//! use icu_timezone::MetazoneCalculator;
+//! use icu::calendar::DateTime;
+//! use icu::timezone::CustomTimeZone;
+//! use icu::timezone::GmtOffset;
+//! use icu::timezone::MetazoneCalculator;
+//! use icu::timezone::IanaToBcp47Mapper;
 //! use tinystr::TinyAsciiStr;
 //!
 //! // Create a time zone for America/Chicago at GMT-6:
 //! let mut time_zone = CustomTimeZone::new_empty();
 //! time_zone.gmt_offset = "-0600".parse::<GmtOffset>().ok();
-//! time_zone.time_zone_id =
-//!     "uschi".parse::<TinyAsciiStr<8>>().ok().map(Into::into);
+//! let mapper = IanaToBcp47Mapper::new();
+//! time_zone.time_zone_id = mapper.as_borrowed().get("America/Chicago");
```

Comment:
maybe keep both examples

```suggestion
//! time_zone.time_zone_id = Some(tinystr!(8, "uschi").into());
//! // Alternatively obtain the BCP47 zone ID from an IANA ID
//! time_zone.time_zone_id = Some(IanaToBcp47Mapper::new().as_borrowed().get("America/Chicago").unwrap());
```

Reply:
Done